shopelectro/selenium/pages/category.py

```diff
@@ -18,11 +20,37 @@ def __init__(self, driver, slug):
     def path(self):
         return reverse('category', args=(self.slug,))
 
-    def product_cards(self) -> typing.List[CatalogCard]:
-        raise NotImplementedError
+    def product_cards(self) -> typing.List[elements.CatalogCard]:
+        products_count = len(self.driver.find_elements(
+            By.CLASS_NAME, 'product-card'
+        ))
+        return [elements.CatalogCard(self.driver, index=i) for i in range(products_count)]
+
+    def find_card(self, id_: int) -> elements.CatalogCard:
+        return elements.CatalogCard(self.driver, id_=id_)
```

Comment:
>id_=id_

Right now we have exposed `Product.id` field in some places to communicate with front.
@duker33  Should we remove it in favor  `Product.vendor_code`? I'd prefer to remove `Product.id` from front totally. It'll get rid of ambiguous for us

Reply:
@artemiy312 , i thought about "id vs vendor_code" problem. I would prefer to save `id`. Let me explain my position.

`vendor_code` comes to us from stock. It should be called `stock_id` in fact. It comes from outside, we don't control it. Tomorrow it can turn from int to str or become not unique or smth. So, we should use it only as an informational code for our users. And should not rely on it to build our infrastructure.

What we need now is responsibility separation between `id` and `vendor_code` throw it interfaces

Reply:
@duker33 I'll create an issue for this discuss